llama.cpp

```diff
@@ -3107,7 +3108,7 @@ static void llm_load_hparams(
                 ml.get_key(LLM_KV_ATTENTION_LAYERNORM_EPS, hparams.f_norm_eps);
                 ml.get_key(LLM_KV_ATTENTION_CAUSAL, hparams.causal_attn);
                 ml.get_key(LLM_KV_TOKENIZER_TOKEN_TYPE_COUNT, hparams.n_vocab_type);
-                ml.get_key(LLM_KV_POOLING_LAYER, hparams.pooling_layer);
+                ml.get_key(LLM_KV_POOLING_TYPE, hparams.pooling_type);
```

Comment:
This is going to break the quants uploaded here:
https://huggingface.co/nomic-ai/nomic-embed-text-v1-GGUF
https://huggingface.co/nomic-ai/nomic-embed-text-v1.5-GGUF

Should we maintain compatibility with existing quants, or is it better to break them to keep the code simple?

Reply:
Yup, was gonna give you the heads up in a minute. I kinda err towards keeping the code simple. Seems like things will be pretty stable after this one change.

Reply:
Agree, better to re-upload the models